Document removal of custom REST wrappers in 8.0 (#83565)
elastic/elasticsearch: docs/reference/migration/migrate_8_0/plugin-changes.asciidoc

they already installed. If any of these plugins are specified in your
configuration file, {es} will ignore them and emit a warning log message.
====

.Third party plugins can no longer intercept REST requests (RestHandlerWrapper)
[%collapsible]
====
*Details* +

> In previous versions of {es}, plugins were able to implement the 
> `getRestHandlerWrapper` method which would allow the plugin to intercept all
> REST requests to the node.  One of the common uses of this feature was to
> implement custom security plugins that replaced the built-in
> {security-features}. This extension point is no longer available to 
> third-party plugins.

Reviewer: ```suggestion
In previous versions of {es}, third-party plugins could implement the 
`getRestHandlerWrapper` method to intercept all REST requests to the node. A 
common use of this feature was to implement custom security plugins to replace
the built-in {security-features}. This extension point is no longer available
to third-party plugins.

```
